console/module/device/src/main/java/org/eclipse/kapua/app/console/module/device/client/device/history/DeviceTabHistory.java

```diff
@@ -97,6 +97,7 @@ public DeviceTabHistory(GwtSession currentSession) {
         super(currentSession, DEVICES_MSGS.tabHistory(), new KapuaIcon(IconSet.HISTORY));
         this.currentSession = currentSession;
         initialized = false;
+        setEnabled(false);
```

Comment:
I don't know if disabling this tab once a Device is deselected is a good idea. Since Events tab doesn't require a communication with the Device itself, and it's only a registry operation, we can just keep it enabled just like any other tab that contains a "slave" grid and shows a "No items" message when no items are selected in a "master" grid.